community-handbook/chaosscast.md

```diff
@@ -202,6 +202,7 @@ These procedures are for the organizer and coordinator of each episode:
   * **Action:** Send out a [confirmation email](https://docs.google.com/document/d/1Pw8OKn-qSSkMw573BZu-EZ4BZualesdZjPbpBEKvzrc/edit) that includes the [“how we podcast” document](https://docs.google.com/document/d/1j5Ng91rrFrx3PdPejv26rBxb1P7UX-O_Q0_I9TCIya0/edit#) to guests and panelists
   * **Action:** Update the episode planning document with recording time and zoom link
   * **Action:** Update the [episode overview table](https://docs.google.com/document/d/1qxmC1k8SayChurIq2r91CR3r_KAThTwaNEMjA6DLGhE/edit#)
+  * **Action:** If the episode is more than two weeks out, schedule a [reminder email](https://docs.google.com/document/d/17vKzCOYvRXq5zaIlv_sb80zFapTU19dRgkx5RqRx3xE/edit) for 1 week before the episode recording
```

Comment:
The reminder template states "tomorrow", not next week. 

This raises the question of whether we should have a 1-week and a 1-day reminder and whether they should look the same or different.

Thoughts?

Reply:
We can just change the reminder email to mention the date and remove the 1-week and 1-day specific reference. Would that work for you?

Reply:
Yes 👍

Reply:
I updated the template